Cargo.toml

```diff
@@ -26,7 +26,7 @@ android_system_properties = "0.1.5"
 core-foundation-sys = "0.8.6"
 
 [target.'cfg(target_os = "windows")'.dependencies]
-windows-core = { version = ">=0.50, <=0.52" }
+windows-core = { version = ">=0.53, <=0.54" }
```

Comment:
are we able to lower the bound on this so we can get the crate building with Rust 1.61.0?

Reply:
Let me do some experimentation to see how far we can stretch this bound _without_ bumping the generator and hence requirements for MSRV/edition, and keep separate PRs open when we do end up bumping those.

Reply:
[`0.53` already requires MSRV `1.62`](https://crates.io/crates/windows-core/0.53.0), the resulting compatibility issues don't even matter at that point.